ee/tabby-ui/lib/hooks/use-copy-to-clipboard.tsx

```diff
@@ -39,15 +27,24 @@ export function useCopyToClipboard({
       navigator.clipboard
```

Comment:
I think it might be better to support passing in a custom `onCopy` function, rather than allowing the function to throw an error directly. Or just modify the implementation of the copy logic inside AssistantMessageCard.

Reply:
Update to use an `onCopyContent` props (avoid div default props onCopy)
tested in vscode and code browser